editor/store/selectors.js

```diff
@@ -1744,3 +1744,22 @@ export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
 export function getEditorSettings( state ) {
 	return state.settings;
 }
+
+/*
+ * Returns the locked state in the context of a given root block.
+ *
+ * @param {Object}  state   Editor state.
+ * @param {?string} rootUID Block UID.
+ *
+ * @return {?string} Locked state in the context of a given block.
+ */
+export function getLockedState( state, rootUID ) {
+	if ( ! rootUID ) {
```

Comment:
Should we make `rootUID` an explicitly optional param so we can avoid the check for falsy values?

```js
getLockedState( state, rootUID = false ) {
  if ( rootUID === false ) {
```

Reply:
Hi @mcsf,
I followed your suggestion with a small change:
```
export function getLockedState( state, rootUID = null ) {
	if ( rootUID === null ) {
```

Sometimes we use other selectors to know the rootUID and they may return null. In that case, we should consider that no rootUID was passed. Using this approach we can avoid extra checks before calling this selector (making the code simpler).

Reply:
I had to revert this change and use the false check because I noticed a regression. Some selectors may return an empty string if rootUID is not set e.g. getBlockRootUID( uid ) returns an empty string for top-level blocks.
The best/simplest way to check for undefined, null or empty string seems to be a false check.